test/TestTransport.php

```diff
@@ -12,18 +12,19 @@ class Castle_RequestTransport
 
   private static $lastRequest = array();
 
-  public function send($method, $url, $body=null, $headers=array()) {
+  public function send($method, $url, $payload) {
     if (empty(self::$params)) {
       self::setResponse(200, '{}');
     }
+    $headers = array('Content-Type' => 'application/json');
```

Comment:
How about sending content length? If I remember correctly, it was needed in som cases

Reply:
Better safe than sorry. Will add :)